tiny_dnn/util/macro.h

```diff
@@ -12,3 +12,10 @@
 #if defined _WIN32 && !defined(__MINGW32__)
 #define CNN_WINDOWS
 #endif
+
+#ifdef _MSC_VER
+#define CNN_MUSTINLINE __forceinline
+#else
+#define CNN_MUSTINLINE __attribute__((always_inline)) inline
```

Comment:
I asked @karandesai-96 to add updated version of macros at #687.
I will rebase this branch once it'll be merged to master.